### PR TITLE
Habitable zone label, improved zooming

### PR DESCRIPTION
**Pull request by nbuonin**
- Implement habitable zone label
- Implement improved zooming
- Production build

### CODE REVIEW

File: circumstellar-habitable-zone-simulator/src/diagram.jsx [hunk 1]
```diff
@@ -105,16 +116,29 @@ export default class CSHZDiagram extends React.Component {
...
+        let pixelsPerAU = this.zoomLevels[0].pixelsPerAU;
+        let pixelsFromStar = this.auToPixels(planetDistance, pixelsPerAU);
         let planetXPosition = STAR_ORIGIN_POINT[0] + pixelsFromStar;
-        while (planetXPosition > ZOOM_UPPER_BREAKPOINT) {
-             zoomLevel++;
-             pixelsFromStar = this.auToPixels(this.props.planetDistance, this.zoomLevels[zoomLevel].pixelsPerAU);
-             planetXPosition = STAR_ORIGIN_POINT[0] + pixelsFromStar;
+        const lastZoomIdx = this.zoomLevels.length - 1;
+        while (planetXPosition > ZOOM_UPPER_BREAKPOINT && pixelsPerAU >= this.zoomLevels[lastZoomIdx].pixelsPerAU) {
+            if (pixelsPerAU < this.zoomLevels[lastZoomIdx].pixelsPerAU) {
```
Comment (sdreher) [inline circumstellar-habitable-zone-simulator/src/diagram.jsx:125]: I think the loop invariant `pixelsPerAU >= this.zoomLevels[lastZoomIdx].pixelsPerAU` eliminates the need for this if statement? But maybe I'm missing something?

**Review by sdreher — COMMENTED** [review]
One code-level comment to review.

The habitable label looks good. I am finding the scale rectangle under the AU label to be a little distracting and confusing. I noticed that the label is centered over the line in the original...maybe that could help?

File: circumstellar-habitable-zone-simulator/src/diagram.jsx (hunk 1 above)
Comment (nbuonin) [reply circumstellar-habitable-zone-simulator/src/diagram.jsx]: Nope didn't miss anything, just me being neurotic.

**Comment by nbuonin** [comment]
I added extra breakpoints to the zoom level indicator to make it less likely to bounce outside of the frame. I think this works a bit better - what do you think? I ran with this because it was easy, but I'm happy to try centering it too.

**Comment by sdreher** [comment]
I also think this works better. Much better.

**Pull request merged**